web/wp-content/mu-plugins/alter-wpcfm-config-path.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Plugin Name: Alter-wpcfm-config-path
```

Comment:
I added this since I noticed `wp-cfm` was installed. I'm okay with the Pantheon plugins, such as PHP sessions and advanced page cache, but am not convinced we should force this on folks.

@stevector is it just left over from copying my other repo?